modules/nlu-testing/src/views/full/index.tsx

```diff
@@ -1,194 +1,123 @@
-import { Button, Icon, Spinner } from '@blueprintjs/core'
-import { AxiosInstance } from 'axios'
-import { Container, SplashScreen } from 'botpress/ui'
-import { toastFailure, toastSuccess } from 'botpress/utils'
-import _ from 'lodash'
-import React from 'react'
-
-import { Test, TestResult } from '../../shared/typings'
-import { computeSummary } from '../../shared/utils'
-
-import { makeApi } from './api'
-import style from './style.scss'
-import { ImportModal } from './ImportModal'
-import { TestModal } from './TestModal'
-import { TestTable } from './TestTable'
-
-interface State {
-  createModalVisible: boolean
-  importModalVisible: boolean
-  tests: Test[]
-  testResults: _.Dictionary<TestResult>
-  loading: boolean
-  working: boolean
-  currentTest?: Test
-}
-
-interface Props {
-  bp: { axios: AxiosInstance }
-  contentLang: string
-}
-
-// TODO use ctx & useReducer instead of state
-export default class NLUTests extends React.Component<Props, State> {
-  private api = makeApi(this.props.bp)
-
-  state: State = {
-    createModalVisible: false,
-    importModalVisible: false,
-    tests: [],
-    testResults: {},
-    loading: true,
-    working: false
+import { Icon, Tab, Tabs } from '@blueprintjs/core'
+import React, { FC, useEffect, useState } from 'react'
+import { FaSkullCrossbones } from 'react-icons/fa'
+import { GiLoad } from 'react-icons/gi'
+import { IoMdCloudDone } from 'react-icons/io'
+import { RiLoader2Line } from 'react-icons/ri'
```

Comment:
Can you find icons in blueprintjs instead ? React-icons is pretty heavy and if we can get rid of it, it would be awesome